llvm/test/CodeGen/AMDGPU/inline-constraints.ll

```diff
@@ -97,39 +97,34 @@ define i32 @inline_A_constraint_H1() {
 
 ; NOSI: error: invalid operand for inline asm constraint 'A'
 ; VI-LABEL: {{^}}inline_A_constraint_H2:
-; VI: v_mov_b32 {{v[0-9]+}}, 0x3c00
```

Comment:
@rampitec I removed those incorrect check lines. I can do it in a separate patch if needed.

Reply:
Still need to check the result.

Reply:
There is no results anymore because the inline asm parser can't know whether `0x3c00` is an inline constant, so the operand doesn't conform with the constraint `A` therefore it will not be parsed further and constructed as an instruction.

Reply:
Can we see that instruction in the test? I.e. having just label checks does not really check anything.

Reply:
No. The instruction is not generated at all.

Reply:
Then this is really broken?

Reply:
Yeah, the test is not correct at the first place.